test/spec/modules/seedtagBidAdapter_spec.js

```diff
@@ -420,6 +428,16 @@ describe('Seedtag Adapter', function() {
         params.adUnitId
       )
     })
+
+    it('shoud set the timeout pixel', function() {
```

Comment:
spelling

is the goal of this PR more stable tests, similar to https://github.com/prebid/Prebid.js/pull/6511/files ? 

If so, much appreciated!

Reply:
I fixed the typo, thanks! With this PR I wanted to fix a bug that we found with the timeout and I wanted to ensure that part was properly tested (that's why I added another test to ensure the pixel is properly set with the right parameters).